doc/classes/RenderingDevice.xml

```diff
@@ -774,6 +774,14 @@
 				Returns [code]true[/code] if the [param texture] is valid, [code]false[/code] otherwise.
 			</description>
 		</method>
+		<method name="texture_native_handle">
+			<return type="int" />
+			<param index="0" name="texture" type="RID" />
+			<description>
+				Returns the internal graphics handle for this texture object. For use when communicating with 3rd party APIs mostly with GDExternal.
```

Comment:
There's still no such thing as GDExternal :)

Please also write it as "third-party".

Reply:
Yeah for some reason that is just stuck in my head :P